### PR TITLE
Move `xdg-empty-trash` inline

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
 'use strict';
+var fs = require('fs');
 var path = require('path');
 var childProcess = require('child_process');
-var xdgEmptyTrash = require('xdg-empty-trash');
 var runApplescript = require('run-applescript');
 var pify = require('pify');
+var Promise = require('pinkie-promise');
```
Comment (sindresorhus) [inline index.js]: You can drop the polyfill usage. Using this module as an API is such an edge I really don't care about Node 0.10 compat.

Comment (kevva) [inline index.js]: Ah, of course. Fixed!

File: index.js [hunk 2]
```diff
@@ -14,5 +16,18 @@ module.exports = function () {
...
+			path.join(dir, 'info')
+		];
+
+		return Promise.all(paths.map(function (pth) {
+			return pify(fs.readdir)(pth).then(function (files) {
+				return Promise.all(files.map(function (file) {
+					return pify(rimraf)(path.join(pth, file));
+				}));
+			});
+		}));
```
Comment (sindresorhus) [inline index.js]: Can you chain the promises instead of nesting them?

Comment (kevva) [inline index.js]: Isn't that going to make it longer?

**Comment by sindresorhus** [comment]
Awesome!

**Comment by sindresorhus** [comment]
Remember to deprecate `xdg-empty-trash` on GitHub and `npm deprecate`.

**Pull request merged**